src/coreclr/nativeaot/BuildIntegration/WindowsAPIs.txt

```diff
@@ -487,7 +487,6 @@ crypt32!CryptGetOIDFunctionValue
 crypt32!CryptHashCertificate
 crypt32!CryptHashCertificate2
 crypt32!CryptHashMessage
-crypt32!CryptHashPublicKeyInfo
```

Comment:
This should be reverted. The purpose of this file is described at the top. It's not "list of APIs we use in the repo".